site/public/js/drag-and-drop.js

```diff
@@ -368,9 +368,187 @@ function validateStudentId(csrf_token, gradeable_id, student_id, submitStudentGr
 }
 
 /**
- *
- * @param submit_url
- * @param return_url
+* @param csrf_token
+* @param gradeable_id
+* @param user_id
+* @param path
+* @param count
+*/
+function moveSubmission(csrf_token, gradeable_id, user_id, path, count) {
+
+    url = buildUrl({'component': 'student', 'page': 'submission', 'action': 'move', 'gradeable_id': gradeable_id});
+
+    var formData = new FormData();
+
+    formData.append('csrf_token', csrf_token);
+    formData.append('user_id', user_id);
+    formData.append('path', path);
+
+    $.ajax({
+        url: url,
+        data: formData,
+        processData: false,
+        contentType: false,
+        type: 'POST',
+        success: function(data) {
+            try {
+                data = JSON.parse(data);
+                if (data['success']) {
+                    $("#bulk_submit_" + count).prop("disabled", true);
+                    $("#bulk_delete_" + count).prop("disabled", true);
+                    $("#bulk_user_id_" + count).prop("disabled", true);
+                    var message ='<div id="submit_' + count +  '" class="inner-message alert alert-success"><a class="fa fa-times message-close" onClick="removeMessagePopup(\'submit_' + count +'\');"></a><i class="fa fa-times-circle"></i>' + data['message'] + '</div>';
+                    $('#messages').append(message);
+                    setTimeout(function() {
+                        $('#submit_' + count).fadeOut();
+                    }, 5000);
+                    return;
+                }
+                else {
+                    alert("ERROR! Please contact administrator with following error:\n\n" + data['message']);
+                }
+            }
+            catch (e) {
+                alert("Error parsing response from server. Please copy the contents of your Javascript Console and " +
+                    "send it to an administrator, as well as what you were doing and what files you were uploading.");
+                console.log(data);
+            }
+        },
+        error: function() {
+            alert("ERROR! Please contact administrator that you could not upload files.");
+        }
+    });
+}
+
+/**
+* @param csrf_token
+* @param gradeable_id
+* @param path
+* @param count
+*/
+function deleteSubmission(csrf_token, gradeable_id, path, count) {
```

Comment:
just a naming comment...  'delete submission' implies deleting a student submission, which we never do.  maybe instead we should refer to it as deleting one item in a bulk pdf, deleteSplitItem, or similar?